science/hacktv/Portfile

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+PortGroup           github 1.0
+
+github.setup        kesterlester hacktv 0.0.4 v
+github.tarball_from releases
+
+categories          science
+platforms           darwin
+license             GPL-3+
+maintainers         kesterlester
+description         Broadcast TV using an SDR like HackRF One
+long_description    Broadcast TV using a Software Defined Radio like HackRF One
+homepage            https://github.com/kesterlester/hacktv
```

Comment:
the homepage will be set automatically by the `github` PG so this line should be removed

Reply:
Done in 48b46f0e0a